privatenetworks_test.go

```diff
@@ -0,0 +1,199 @@
+package glesys
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPrivateNetworksCreate(t *testing.T) {
+	c := &mockClient{body: `{ "response": { "privatenetwork":
+		{ "id": "pn-123ab", "name": "mynetwork", "ipv6aggregate": "fe00:123::/48"}}}`}
```

Comment:
```suggestion
		{ "id": "pn-123ab", "name": "mynetwork", "ipv6aggregate": "2001:db8::/48"}}}`}
```

Use RFC3849 documentation network (as long as no validation is done client side but I think we don't have this here).

Same basic problem exists in multiple places ofc =)